index.html

```diff
@@ -224,5 +224,44 @@ <h3>LET'S CONNECT</h3>
         </ul>
       </section>
     </section>
+    <footer class="form-footer">
+      <h2>Contact me</h2>
+      <p>
+        If you have an application you are interested in developing, a feature
+        that you need built or a project that needs coding. I'd love to help
+        with it
+      </p>
+      <form
+        class="form-container"
+        action="https://formspree.io/f/{form_id}"
+        method="post"
+      >
+        <input
+          type="text"
+          name="name"
+          id="name"
+          placeholder="Full name"
+          maxlength="30"
+          required
+        />
+        <input
+          type="email"
+          name="email"
+          id="email"
+          placeholder="Email address"
+          required
+        />
+        <textarea
+          name="message"
+          id="message"
+          rows="7"
+          placeholder="Write your message here"
+          maxlength="500"
+          required
+        ></textarea>
+        <button class="contact-form-button" type="submit">Get in touch</button>
+        <span class="bottom-line"></span>
+      </form>
```

Comment:
- [ ] You did a great job working on this milestone! 💯 However, you are not using the `formspree` service the way it is expected. I'm getting this screen when I try to submit the form:

![image](https://user-images.githubusercontent.com/77069011/220469413-27833a0e-a06a-4212-b293-841d6f08208f.png)

This happens because, in your form action, you have the `{form_id}` at the end. This value uses curly braces because it's supposed to be dynamic, this form_id should be the `id` you created in [`formspree`](https://formspree.io/html/) so please consider fixing this and testing it before submitting your code again! 🤓 🔧


- [ ] I'm also getting a whitespace at the right edge of your page that is making the design look buggy. This is caused by your about section, please consider fixing this! 🤓 🔧 

|Template|Your design|
|---|---|
| ![image](https://user-images.githubusercontent.com/77069011/220470129-24a0f408-606a-4c36-ac52-e66784bce457.png) | ![image](https://user-images.githubusercontent.com/77069011/220470070-a8eab25d-8628-4ede-ba04-70cc7c92a3cd.png) |